src/containers/auth/InviteScreen.js

```diff
@@ -12,10 +12,14 @@ export default class InviteScreen extends Component {
 
   render() {
     const { actions } = this.props;
+
     return (
-      <Invite
-        onSubmit={actions.user.invite}
-      />
+      <div className="auth__container auth__container--signup">
```

Comment:
i know: presentational stuff in containers, however I wanted to reuse the dumb component `Invite` for both screens

Reply:
@haraldox this would make navigating through the code unecessary complicated, please move this back to the presentational component with a condition.

Reply:
@adlk done.